vassal-app/src/main/java/VASSAL/counters/PieceDefiner.java

```diff
@@ -1281,15 +1292,36 @@ public Component getListCellRendererComponent(
       // It is incredibly inefficient for GamePieces and is not needed
       // since we overwrite the label text anyway.
       super.getListCellRendererComponent(list, "", index, selected, hasFocus);
+
+      final int lineOffset = definer.isPrototype() ? 1 : 0;
+
       if (value instanceof EditablePiece) {
-        setText(((EditablePiece) value).getDescription());
+        final EditablePiece editableValue = (EditablePiece) value;
+        final String type = editableValue.getType().split(";")[0] + ";";
```

Comment:
Why is this needed? You can just check for if (editableValue.getType().startsWith(Comment.ID)). No need to break it up since the ; is already included in the ID.

Reply:
Thanks - fixed in next build.